DEVELOPMENT.md

```diff
@@ -460,6 +472,18 @@ For this we need:
 Once you've created the _Developer ID Application_ certificate, download it
 locally and add it to your keychain by double clicking on the file.
 
+Before building a notarized DMG, make sure you're connected to the internet
+and then run:
+
+```sh
+    git checkout vX.X.X
```

Comment:
The extra indentation leads to extra spaces in the code block. Could you remove it?

Reply:
62d21f1f7a7e0d3cebe5d8d3844535fadf9d9036